src/otx/core/metrics/kitti_3d_eval/eval.py

```diff
@@ -65,12 +33,12 @@ def clean_data(
         difficulty (int): The difficulty level.
 
     Returns:
-        tuple: The number of valid objects, ignored_gt, ignored_dt, and dc_bboxes.
+        tuple: The number of valid objects, ignored_gt, ignored_dt.
     """
-    min_height = [40, 25, 25]
-    max_occlusion = [0, 1, 2]
-    max_truncation = [0.15, 0.3, 0.5]
-    dc_bboxes, ignored_gt, ignored_dt = [], [], []
+    min_height = 20
```

Comment:
According to value below, it should be set as 25. Is it intended to set it to 20?

Reply:
I decided to soft the limits of data filtering and keep more boxes. Previously, we had so called difficulties defined by KITTI benchmark, I removed them now and keep only one number to filter really bad GTs and predictions. I also removed truncation and occlusion, since this info is not available in other datasets and Geti.